resolwe/flow/executors/__init__.py

```diff
@@ -122,6 +123,14 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
+        # Fetch data instance to get any executor requirements.
+        if data_id != 'no_data_id':
```

Comment:
Yes, this is used during some tests, which do a lot of mocking. I don't like it, but without this, these other tests will fail.